src/PantherTestCaseTrait.php

```diff
@@ -80,6 +80,7 @@ trait PantherTestCaseTrait
         'router' => '',
         'external_base_uri' => null,
         'readinessPath' => '',
+        'browser' => self::CHROME,
```

Comment:
Why not firefox as base? :fox_face:

Reply:
Because it's a huge BC break, and because Chrome is more used (unfortunately): 64% for Chrome, 5% for Firefox.